### PR TITLE
Fixes Ierde's allergy to their own spell.

### PR DESCRIPTION
**Pull request by CwasCard**
#### Summary
Mods "Fixes Ierde's iron allergy to their spellcasted armor."

#### Purpose of change

Ierde have an allergy to their Invulnerable Iron Skin mutation. This changes that.

#### Describe the solution

Changes the material of their Invulnerable Iron Skin spell from iron to Moon's Tears, the fae equivalent of iron. They will not suffer their iron/steel allergy as a result.

#### Describe alternatives you've considered

Letting Ierde stay allergic to their own magic.

#### Testing

Copy-pasted changes into my own game, loaded up a fresh world with just DDA & Xedra Evolved, gave myself the mutation that gives me the spell, cast the spell, no more allergy.

#### Additional context
Screenshot 2024-09-18 181217

image

i like paraclesians

### CODE REVIEW

**Comment by Maleclypse** [comment]
Double checking what are the protection values under the allergy inducing metal.

**Comment by Standing-Storm** [comment]
Yeah, the thickness might need to be reduced if moon's tears is that much better

**Comment by CwasCard** [comment]
> Yeah, the thickness might need to be reduced if moon's tears is that much better

Alrighty, switched the thickness to 1.2mm.

Stats should be as follows now:
image

File: data/mods/Xedra_Evolved/items/ethereal_items.json [hunk 1]
```diff
@@ -385,14 +385,14 @@
...
+    "description": "Fae iron essence.",
     "weight": "0 g",
     "volume": "0 L",
-    "material": [ "iron" ],
+    "material": [ "moon_tears" ],
     "symbol": "[",
     "color": "dark_gray",
     "warmth": 5,
-    "material_thickness": 5,
+    "material_thickness": 1.2,
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/items/ethereal_items.json:395]:
```suggestion
    "material_thickness": .6,
```

**Pull request merged**